Cargo.toml

```diff
@@ -17,7 +17,9 @@ members = [
     "wgpu-types",
     "wgpu",
 ]
-exclude = []
+exclude = [
+    "d3d12",
```

Comment:
Why are we excluding d3d, this should be a default member of the repo?

Reply:
Per previous conversation [here](https://github.com/gfx-rs/wgpu/pull/3987#issuecomment-1728471456), this is to avoid breakage with WASM builds (since all members of the workspace are attempted to be built, but we don't actually want that in WASM builds).

Reply:
Okay lets fix this in a follow up